### DISCUSSION TITLE
Formule 1 API losing developer, issue for the card?

### DISCUSSION THREAD

**Discussion by B1rdEater** [marcokreeft87/formulaone-card#115] [Announcements]
Recently the API developer wrote he will not be pursuing the project. Will this be a problem for this card?

"Unfortunately, I will not be pursuing the project. It takes to much of my time and ultimately brakes my company's web server. I will be shutting down the project on February 15th if nobody takes over"

https://github.com/delzear/hass-formulaoneapi/issues/71

**Comment by marcokreeft87** [comment]
The card isnt dependent on the api anymore, so no :)

**Comment by B1rdEater** [comment]
Glad to hear :)! Well done 👍
